.github/workflows/conformance.yml

```diff
@@ -0,0 +1,292 @@
+name: Talos Conformance
+on:
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - reopened
+  schedule:
+    # Run weekly.
+    - cron: '* * * * 1'
```

Comment:
Shouldn't this be something like `0 0 * * 1`? Otherwise it's "At **every** minute on Monday."